### DISCUSSION TITLE
uint16 gf6

### DISCUSSION THREAD

**Discussion by jxhoh** [opengeos/segment-geospatial#143] [Q&A] [answered]
能支持对unit16位图像的数据分割吗，例如gf6

**Comment by giswqs** [comment, marked as answer]
Not supported at the moment.

**Reply by Surv-Lukmon** [reply]
@giswqs  I would like to know the data types that are supported for satellite imagery. I am trying out the Segment Anything Model using the PlanetScope image in **uint16**, but I keep getting the error below:

  333 
    334     if mode is None:
--> 335         raise TypeError(f"Input type {npimg.dtype} is not supported")
    336 
    337     return Image.fromarray(npimg, mode=mode)

TypeError: Input type uint16 is not supported
